prow/config/config.go

```diff
@@ -215,8 +215,23 @@ type Sinker struct {
 	MaxPodAge time.Duration `json:"-"`
 }
 
+// Spyglass holds config for Spyglass
+type Spyglass struct {
+	// Viewers is a map of Regexp strings to viewer names that defines which sets
+	// of artifacts need to be consumed by which viewers
+	Viewers map[string][]string `json:"viewers,omitempty"`
+	// SizeLimit is the max size artifact in bytes that Spyglass will attempt to
+	// read in entirety. This will only affect viewers attempting to use
+	// artifact.ReadAll(). To exclude outlier artifacts, set this limit to
+	// expected file size + variance. To include all artifacts with high
+	// probability, use 2*maximum observed artifact size.
```

Comment:
Does this assume a normal distribution of artifact sizes? ;)

Reply:
It does. If you'd like to collect some metrics on test artifact sizes that would be great! Until we have that data (and perhaps even with that data...) I'd prefer to mention only that `size_limit` is dependent on both your use case and the distribution of your test artifact sizes. Unfortunately that gives no real concrete advice to the administrator of the system, so I settled on this.